src/Compilers/Core/Portable/Operations/IRangeOperation.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    /// <summary>
+    /// Represents a range operation.
+    /// <para>
+    /// Current Usage:
+    ///  (1) C# range expressions
+    /// </para>
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    public interface IRangeOperation : IOperation
+    {
+        /// <summary>
+        /// Left operand.
+        /// </summary>
+        IOperation LeftOperand { get; }
+
+        /// <summary>
+        /// Right operand.
+        /// </summary>
+        IOperation RightOperand { get; }
+
+        /// <summary>
+        /// <code>true</code> if this is a 'lifted' range operation.  When there is an 
+        /// operator that is defined to work on a value type, 'lifted' operators are 
+        /// created to work on the <see cref="System.Nullable{T}"/> versions of those
+        /// value types.
+        /// </summary>
+        bool IsLifted { get; }
```

Comment:
>IsLifted [](start = 13, length = 8)

Is lifting applicable to a range operator?

Reply:
Yes. Please see LocalRewriter_Range.cs

---
In reply to: [185130307](https://github.com/dotnet/roslyn/pull/26405#discussion_r185130307) [](ancestors = 185130307)